lib/github_collaborators/terraform_collaborator.rb

```diff
@@ -129,4 +129,54 @@ def get_value(val)
       nil
     end
   end
+
+  # Most of the functionality within this class is file based
+  # Unlike the GithubCollaborator functionality which can be pointed else where
+  # If you wish to use this elsewhere, the code must be copied to the repo with
+  # The target terraform/*.tf files
+  class TerraformCollaborators
+    attr_reader :base_url
```

Comment:
Do we have any tests setup for this sort of thing? I am not very familiar with the scope of the test suite, but it would be good to add some if the setup is already there.